src/services/backendService/reducer.js

```diff
@@ -0,0 +1,52 @@
+import {
+  FETCH_FROM_BACKEND_FAILED,
+  FETCH_FROM_BACKEND_IS_ON,
+  FETCH_FROM_BACKEND_SUCCESS
+} from "./actionTypes";
+import * as R from "ramda";
+import moment from "moment";
+
+function getNextState(key, subKey, _path, data, state) {
+  const path = _path || [key, subKey].filter(Boolean);
+  return R.assocPath(path, data, state);
+}
+
+export default function(state, action) {
+  switch (action.type) {
+    case FETCH_FROM_BACKEND_FAILED:
+      return getNextState(
+        action.key,
+        action.subKey,
+        action.path,
+        {
+          status: "erroneous"
```

Comment:
It would be more informative to use real constants instead of matching string literals
(comparisons at least in backendService/index.js)

Reply:
Very good point. Will be fixed.